src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldDecimal.class.php

```diff
@@ -9,10 +9,9 @@
  * file that was distributed with this source code.
  */
 
-require_once PATH_LIBRARY.'/functions/ConvertNumber.fun.php';
 /**
  * a number (int).
-/**/
+ * /**/
```

Comment:
```suggestion
 */
```

Reply:
Whoops? Where did this come from. Fixed.